### ISSUE TITLE
Support specify timeout for Milvus()

### ISSUE THREAD

**Issue by yhmo** [milvus-io/pymilvus#539]
**Describe the bug**
The Milvus() will wait for 90 seconds if milvus server is not running. The Milvus() call to ping(), the ping() has a parameter "timeout=30", and the GrpcHandler has a member "max_retry", its default value is 3. After 90 seconds, the Milvus() will raise an exception "Fail connecting to server on xxx. Timeout"
This behavior is not friendly to users. Maybe the Milvus() should provide a "timeout" parameter.

**Expected behavior**
Set timeout for Milvus() so we can write code like this:

```
try:
    milvus = Milvus(host=SERVER_ADDR, port=SERVER_PORT, timeout=3)
except:
    LOGGER.error("Connect server {} failed.".format(SERVER_ADDR))
    raise
```

**Comment**
This issue makes users hard to apply pymilvus in production.

**Comment by anitas49** [comment]
just curious, if this can be included in version 1.1.x pymilvus? thanks!

**Comment by XuanYang-cn** [comment]
> just curious, if this can be included in version 1.1.x pymilvus? thanks!

@anitas49 Yes, will be in pymilvus v1.1.2

**Comment by XuanYang-cn** [comment]
pymilvus v1.1.2 released
